keymaps/ava.cson

```diff
@@ -0,0 +1,3 @@
+'atom-workspace':
+  'ctrl-alt-a': 'ava:toggle'
+  'ctrl-alt-x': 'ava:run'
```

Comment:
There is no big reason for having two today. I used two because they allowed me to easily test in isolation the two steps. I'll compact them in one. 

If in the future we have a parsing of the test file (something that I'd like to discuss after having a nice stable first version) phase we'll need two. No need to worry about this now.